jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -650,20 +650,20 @@ static irep_idt get_if_cmp_operator(const irep_idt &stmt)
 /// Build a member exprt for accessing a specific field that may come from a
 /// base class.
 /// \param pointer: The expression to access the field on.
-/// \param fieldref: A getfield/setfield instruction produced by the bytecode
+/// \param field_instruction: A getfield/setfield instruction produced by the bytecode
 /// parser.
 /// \param ns: Global namespace
 /// \return A member expression accessing the field, through base class
 /// components if necessary.
 static member_exprt
-to_member(const exprt &pointer, const exprt &fieldref, const namespacet &ns)
+to_member(const exprt &pointer, const exprt &field_instruction, const namespacet &ns)
 {
-  struct_tag_typet class_type(fieldref.get(ID_class));
+  struct_tag_typet class_type(field_instruction.get(ID_class));
 
   const exprt typed_pointer =
     typecast_exprt::conditional_cast(pointer, java_reference_type(class_type));
 
-  const irep_idt &component_name = fieldref.get(ID_component_name);
+  const irep_idt &component_name = field_instruction.get(ID_component_name);
```

Comment:
Is there a `get_component_name` that could be used instead?

Reply:
Thanks for persisting with this, I see there is a `fieldref_exprt`. To do it properly would take a little time, would you like to get this PR in first as a partial improvement and then I'll follow up with using fieldref_exprt more comprehensively.

Reply:
either way is fine for me